### PR TITLE
update generator.py

### PR DESCRIPTION
**Pull request by ceyusa**
I just started to hack on this and found useful generator.py to understand the process. But it seems a bit obsolete. These two patches tries to update the usage of gir on its master branch.

### CODE REVIEW

File: generator.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def update_workspace():
 for entry in [f for f in listdir('.') if isfile(join('.', f))]:
     if entry.startswith('Gir_Gst') and entry.endswith('.toml'):
         print('==> Regenerating "{}"...'.format(entry))
-        call(['./target/release/gir', '-c', entry])
+        call(['./target/release/gir', '-c', entry, '-d', 'gir-files', '-m', 'sys'])
```
Comment (ceyusa) [inline generator.py]: just realize this patch is not needed since all this data is in the toml file

Comment (sdroege) [reply generator.py]: Yeah, can you drop that? :)

Comment (ceyusa) [reply generator.py]: done ;)

**Comment by sdroege** [comment]
Thanks :)

**Pull request merged**